modules/upload/upload.service.js

```diff
@@ -0,0 +1,119 @@
+require('dotenv').config();
+const azureStorage = require('azure-storage');
+const blobService = azureStorage.createBlobService(
+    process.env.STORAGE_ACCOUNT, 
+    process.env.ACCESS_KEY, 
+    process.env.AZURE_HOST
+);
+const containerName = 'images';
+const getStream = require('into-stream');
+const Jimp = require("jimp");
+const uniqid = require('uniqid');
+
+class UploadService {
+    async uploadFiles(files) {
+        return files.map(async file => {
+            const { createReadStream, filename } = await file;
```

Comment:
Do we need an `await` here? Is `file` a promise?

Reply:
File on Upload type is promise